README.md

```diff
@@ -122,115 +69,57 @@ After you're finished please remove all the comments and instructions!
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
```

Comment:
[OPTIONAL]
- [x] Please kindly remove these information as they only to guide you in customizing this section :+1: :sparkles: